config/v1/types_node.go

```diff
@@ -42,6 +42,14 @@ type NodeSpec struct {
 	// the status and corresponding reaction of the cluster
 	// +optional
 	WorkerLatencyProfile WorkerLatencyProfileType `json:"workerLatencyProfile,omitempty"`
+
+	// eventedPleg enables the event based PLEG between the kubelet and CRI-O
+	// Reference: https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/3386-kubelet-evented-pleg/README.md
+	// Valid values are `Enabled`, `Disabled` and ""
+	// By default, the evented pleg feature is not enabled in the cluster
```

Comment:
We typically have a particular prose we use for this, can you update please
```suggestion
	// When omitted, this means no opinion and the platform is left to choose a reasonable default, which is subject to change over time.
	// The current default value is Disabled.
```